modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -60,97 +36,91 @@ output application/dw
 ----
 * In DataWeave 2.3.0, the expression returns `PT8952H`.
 * In DataWeave 2.2.0, the expression returns `P1Y8D`.
-
-
+* Type: `Boolean`
```

Comment:
The type and default bullets are in the wrong place. Needs to be fixed in the source.

Reply:
@andres-rad yes, before the example.

Reply:
fixed in PR